src/amo/components/AddonMoreInfo/index.js

```diff
@@ -9,33 +9,55 @@ import Link from 'amo/components/Link';
 import { getVersionById, getVersionInfo } from 'core/reducers/versions';
 import { STATS_VIEW } from 'core/constants';
 import translate from 'core/i18n/translate';
+import { fetchCategories, getCategoryNames } from 'core/reducers/categories';
 import { hasPermission } from 'amo/reducers/users';
-import type { AddonType } from 'core/types/addons';
 import { isAddonAuthor, trimAndAddProtocolToUrl } from 'core/utils';
 import Card from 'ui/components/Card';
 import DefinitionList, { Definition } from 'ui/components/DefinitionList';
 import LoadingText from 'ui/components/LoadingText';
 import { addQueryParams } from 'core/utils/url';
 import type { AppState } from 'amo/store';
+import type { ErrorHandlerType } from 'core/errorHandler';
 import type { AddonVersionType, VersionInfoType } from 'core/reducers/versions';
+import type { AddonType } from 'core/types/addons';
 import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
 import type { ReactRouterLocationType } from 'core/types/router';
 
 type Props = {|
   addon: AddonType | null,
   i18n: I18nType,
+  relatedCategories: Array<string> | null,
```

Comment:
Sure! I will update this.

Reply:
No, it should definitely be in `InternalProps`, just like the other props that are being added via `mapStateToProps`, e.g., `currentVersion`. If you're getting a Flow error it might be due to some of the other changes I suggest you make around `relatedCategories`. The best thing is to make all of the changes I suggested, and if you're still getting Flow errors push a new commit and I can help you debug them.

Reply:
Ohk! I will create a commit with the change.